src/applications/ivc-champva/10-7959f-2/config/form.js

```diff
@@ -17,6 +17,8 @@ import {
   yesNoSchema,
 } from 'platform/forms-system/src/js/web-component-patterns';
 
+import FileFieldCustom from '../../shared/components/fileUploads/FileUpload';
+
```

Comment:
We can skip using the `FileFieldCustom` for this form - this is what lets us use the `MissingFileOverview`, which I don't think we need for F-2?